pycon/settings.py

```diff
@@ -38,39 +39,28 @@
         del os.environ['HTTPS']
     HTTPS = False
 
-#from django.utils.translation import ugettext as _
-_ = lambda x:x
-
 ADMINS = (
     ('web-wg', 'web-wg@europython.eu'),
 )
-
 MANAGERS = ADMINS
 
-ALLOWED_HOSTS = os.environ.get("ALLOWED_HOSTS", '*').split(',')
-#APPEND_SLASH = False
+ALLOWED_HOSTS = config('ALLOWED_HOSTS', default='*', cast=lambda v: [s.strip() for s in v.split(',')])
+APPEND_SLASH = config('APPEND_SLASH', default=False, cast=bool)
 
-PROJECT_DIR = os.environ.get('PROJECT_DIR', os.path.normpath(
-    os.path.join(os.path.dirname(__file__), '..')))
-DATA_DIR = os.environ.get('DATA_DIR', os.path.join(PROJECT_DIR, 'data'))
-OTHER_STUFF = os.environ.get('OTHER_STUFF',
-                             os.path.join(PROJECT_DIR, 'documents'))
+PROJECT_DIR = config('PROJECT_DIR', default=os.path.normpath(os.path.join(os.path.dirname(__file__), '..')))
+DATA_DIR = config('DATA_DIR', default=os.path.join(PROJECT_DIR, 'data'))
+OTHER_STUFF = config('OTHER_STUFF', default=os.path.join(PROJECT_DIR, 'documents'))
 
 LOGS_DIR = os.path.join(PROJECT_DIR, 'logs/')
 
-if not os.path.exists(LOGS_DIR):
-    os.makedirs(LOGS_DIR)
-
-sys.path.insert(0, os.path.join(PROJECT_DIR, 'deps'))
-
 SITE_DATA_ROOT = DATA_DIR + '/site'
 
-DATABASE_TYPE = os.environ.get('DATABASE')
+DATABASE_TYPE = config('DATABASE', default='')
 
 if DATABASE_TYPE == "postgres":
```

Comment:
this can be dropped for now, and later, if needed, we can probably go with dj-database-url anyway.